requirements.txt

```diff
@@ -2,3 +2,4 @@ lxml
 six
 futures
 paramiko
+twine
```

Comment:
This probably shouldn't be in the `requirements.txt` since it isn't a requirement for running the package, just for publishing it